compose/compose_oauth2.go

```diff
@@ -118,3 +118,21 @@ func OAuth2ResourceOwnerPasswordCredentialsFactory(config *Config, storage inter
 		},
 	}
 }
+
+// OAuth2TokenRevocationFactory creates an OAuth2 token revocation handler and registers
+// an access token, refresh token and authorize code validator.
+func OAuth2TokenRevocationFactory(config *Config, storage interface{}, strategy interface{}) interface{} {
+	return &struct {
+		*oauth2.TokenRevocationHandler
+		*oauth2.CoreValidator
```

Comment:
The `CoreValidator` looks like he doesn't belong here. This is probably something that the compose package requires, right? I have to check if we can remove this requirement.